modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/GridExchangeFreeSwitchTest.java

```diff
@@ -159,6 +161,22 @@ public void testBaselineNodeLeftOnFullyRebalancedCluster() throws Exception {
         }
     }
 
+    /**
+     * Checks Partition Exchange is regular in case of fixed baseline.
+     * When IGNITE_PME_EXCHANGE_FREE is False.
+     */
+    @Test
+    public void testBaselineNodeLeftOnFullyRebalancedClusterPmeExchangeFreeFalse() throws Exception {
+        System.setProperty(IGNITE_PME_EXCHANGE_FREE, Boolean.FALSE.toString());
```

Comment:
Should we use IgniteSystemProperty class?
Should we use *boolean method instead of toStringing?

Reply:
class IgniteSystemProperties does not have set methods.

Reply:
can we just use "true"?